<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QMainWindow" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PictureIcon">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="connectionLabelStatus">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Downloads/LogoScopic(1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="schoolLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/Diploma-Certificate-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="majorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/Science-School-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="noteSpaceLogo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/Icons/notes_mainLabel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="schoolLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="majorLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/pics/Icons/search_files.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>PictureIcon</zorder>
   <zorder>connectionLabelStatus</zorder>
   <zorder>infoLabel</zorder>
   <zorder>schoolLabel</zorder>
   <zorder>majorLabel</zorder>
   <zorder>noteSpaceLogo</zorder>
   <zorder>userNameLine</zorder>
   <zorder>schoolLine</zorder>
   <zorder>userNameLabel</zorder>
   <zorder>majorLine</zorder>
   <zorder>load</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Note"/>
    <addaction name="actionAdd_Note"/>
    <addaction name="actionView_Notes"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_Note"/>
   <addaction name="actionAdd_Note"/>
   <addaction name="actionView_Notes"/>
   <addaction name="actionLog_Out"/>
  </widget>
  <action name="actionAdd_Note">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/Icons/new-file-icon.png</normaloff>
     <normalon>Icons/Document-Write-icon.png</normalon>:/pics/Icons/new-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
  </action>
  <action name="actionView_Notes">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/Icons/search_files.png</normaloff>:/pics/Icons/search_files.png</iconset>
   </property>
   <property name="text">
    <string>View Notes</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/logout-icon.png</normaloff>:/Icons/logout-icon.png</iconset>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionCreate_Note">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Document-Write-icon.png</normaloff>:/Icons/Document-Write-icon.png</iconset>
   </property>
   <property name="text">
    <string>Create Note</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
